orchagent/pfc_detect_mellanox.lua

```diff
@@ -78,27 +80,30 @@ for i = n, 1, -1 do
                         packets_last = tonumber(packets_last)
                         pfc_rx_packets_last = tonumber(pfc_rx_packets_last)
                         pfc_duration_last = tonumber(pfc_duration_last)
-                        local storm_condition = (pfc_duration - pfc_duration_last) > (poll_time * 0.8)
+                        local storm_condition = (pfc_duration - pfc_duration_last) > (effective_poll_time * 0.99)
 
                         -- Check actual condition of queue being in PFC storm
-                        if (occupancy_bytes > 0 and packets - packets_last == 0 and pfc_rx_packets - pfc_rx_packets_last > 0) or
+                        if (occupancy_bytes > 0 and packets - packets_last == 0 and storm_condition) or
```

Comment:
Since `pfc_rx_packets` and `pfc_rx_packets_last` are not used any longer, is it still necessary to read it from db?

Reply:
I suggest keeping it for debug proposes.